src/js/components/PodDetail.js

```diff
@@ -21,14 +28,41 @@ class PodDetail extends mixin(TabsMixin) {
     };
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift()
+      currentTab: Object.keys(this.tabs_tabs).shift(),
+      activeActionDialog: null
     };
 
     METHODS_TO_BIND.forEach((method) => {
       this[method] = this[method].bind(this);
     });
   }
 
+  handleAction(action) {
+    switch (action.id) {
```

Comment:
Not a big fan of using an object arg with a single parameter here, it feels like more complexity than we need. 

If we made the action type the first name, we could bind in the JSX like
```onClick={props.handleAction.bind(null, PodActionItem.DESTROY)}```

But I'd actually prefer individual explicit handleDestroy/handleEdit/handleScale/handleSuspend  methods.